exercises/Functions-6-R.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: exercise
+title: Functions 6
+language: R
+---
+
+This is a follow up to [Strings 6]({{ site.baseurl }}/exercises/Strings-6/).
+
+A colleague has produced a file with one DNA sequence on each line. Download
+[the file]({{ site.baseurl }}/data/dna_sequences_1.txt) and load it into R using
+`read.csv()`. The file has no header and is separated by white space (`’’`).
```

Comment:
>> lets us get rid of the extra sep language
Done.

>> Instead of the * what about just add the description immediately
following the original mention:
Removed * and rewrote paragraph for clarity.

>> Does this need to be Strings-6-R
Done.

On Mon, May 4, 2015 at 5:36 PM, Ethan White <notifications@github.com>
wrote:

> In exercises/Functions-6-R.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/36#discussion_r29630793>
> :
>
> > @@ -0,0 +1,22 @@
> > +---
> > +layout: exercise
> > +title: Functions 6
> > +language: R
> > +---
> > +
> > +This is a follow up to [Strings 6]({{ site.baseurl }}/exercises/Strings-6/).
> > +
> > +A colleague has produced a file with one DNA sequence on each line. Download
> > +[the file]({{ site.baseurl }}/data/dna_sequences_1.txt) and load it into R using
> > +`read.csv()`. The file has no header and is separated by white space (`’’`).
>
> Yeah, I think there is a space delimited file somewhere that needs this.
> In this case since there aren't separators on each line you could actually
> use anything for sep, so I think it's clearer and lets us get rid of the
> extra language to just not use it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/36/files#r29630793>
> .
>